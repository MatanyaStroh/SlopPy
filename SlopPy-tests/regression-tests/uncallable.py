s = 'hello'
x = s()
assert type(x) is NA
