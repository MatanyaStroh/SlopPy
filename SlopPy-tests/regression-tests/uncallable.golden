{NA <type 'exceptions.TypeError'> "'str' object is not callable"
 Traceback (most recent call last):
  File "regression-tests/uncallable.py", line 2, in <module>
    x = s()

Locals: {'__builtins__': <module '__builtin__' (built-in)>, '__file__': 'regression-tests/uncallable.py', '__package__': None, 's': 'hello', '__name__': '__main__', '__doc__': None}
}
